linuxkit/linuxkit
# Kernel push builder

**- What I did**

Fixed the `kernel/Makefile` target `push`. It worked for kernels and tools, but failed on tagging the `*-builder` correctly.

This now uses the recently added `lkt pkg remote-tag` to tag it.

**- How I did it**

Changed the `kernel/Makefile`

**- How to verify it**

First, run it manually and check; I did.
Second, it is used in CI on merge-to-master.

**- Description for the changelog**

fixed `make -C kernel push`
